shared/common-adapters/back-button.native.js

```diff
@@ -19,10 +20,12 @@ export default class BackButton extends Component {
 
   render () {
     return (
-      <Box style={{...styles.container, ...this.props.style}} onClick={e => this.onClick(e)}>
```

Comment:
onClick -> onPress and you can't add a handler like that on a raw View/Box, have to use a Touchable* view